### PR TITLE
@craigspaeth => Adds tracking to footer markdown links

### PR DESCRIPTION
**Pull request by kanaabe**
cc @anipetrov for naming

Closes https://github.com/artsy/publishing/issues/35

### CODE REVIEW

**Comment by kanaabe** [comment]
@owendodd we can search by `destination_path` for the url that was clicked

File: desktop/components/main_layout/templates/index.jade [hunk 1]
```diff
@@ -17,7 +17,7 @@ html(
       
     //- Don't include fonts if Reflection is requesting it		
     if sd.BROWSER && sd.BROWSER.family != 'PhantomJS'
-      link( type='text/css', rel='stylesheet', href='#{sd.WEBFONT_URL}/force-webfonts.css')
+      link( type='text/css', rel='stylesheet', href='#{sd.WEBFONT_URL}force-webfonts.css')
```
Comment (craigspaeth) [inline desktop/components/main_layout/templates/index.jade]: Works for me, but this trailing slash could probably just be removed in the environment variable?

**Comment by craigspaeth** [comment]
LGTM! Minor non-blocking comment, feel free to merge when 💚

File: desktop/components/main_layout/templates/index.jade (hunk 1 above)
Comment (kanaabe) [inline desktop/components/main_layout/templates/index.jade]: You're right, it will look better too 👍  Updating!

Comment (kanaabe) [inline desktop/components/main_layout/templates/index.jade]: Actually this was just a problem with my .env anyway so no change necessary. I should have been more precise!

**Pull request merged**